script/lib/application_insights.py

```diff
@@ -24,6 +24,8 @@ def single_domain_event(event_type):
          | where cloud_RoleName in ('hmpps-domain-event-logger')
          | where name == '{event_type}'
          | where timestamp between((ago(7d)) .. now())
+         | order by timestamp desc
```

Comment:
ensure that we get the latest event in case the format has changed in last 7 days